test/nettle.jl

```diff
@@ -0,0 +1,32 @@
+import Nettle
+
+function test_equal_state(m::MD5.MD5_CTX,n::Nettle.Hasher)
+    @test (n.state)[1:16] == reinterpret(UInt8, m.state)
+end
+
+@testset "Against Nettle initialize, update!, digest!" begin
+    for chunkcount ∈ [0,1,2,3, 10, 100]
```

Comment:
My personal style is to use `in` in forloops,
and reserve  `∈` for operations on sets.

and since the loop later uses `in` this should also for consistancy